### PR TITLE
Fix SendGrid and GitHub typos

### PR DESCRIPTION
**Pull request by hugovk**
### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [n/a] I have added tests that prove my fix is effective or that my feature works
- [n/a] I have added necessary documentation about the functionality in the appropriate .md file
- [n/a] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Fix SendGrid typo
- Fix GitHub typo
- Fix some other typos
- Fix some inconsistent fullstops

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ print(header.json_string())
 * [Categories](#categories)
 * [Schedulling Parameters](#schedulling-parameters)
```
Comment (42B) [inline USAGE.md]: Typo? #88 also fixes this typo but only the second occurrence on line 155. With one or two extra commits, this PR could make #88 redundant.

Comment (hugovk) [reply USAGE.md]: Thanks, I'll suggest the fixes in #88.

**Comment by hugovk** [comment]
Merge conflict resolved.

**Comment by hugovk** [comment]
Hello, is there anything extra I can do for this PR? Thanks!

**Comment by hugovk** [comment]
It's nearly time for the next Hacktoberfest! This PR has been approved, is anything else needed to merge it? Thanks!

**Comment by hugovk** [comment]
Closing these year-old PRs. Let me know if you're still interested and I can re-open. Thanks!

**Pull request closed without merging**